extensions/azurePublishNew/src/components/resourceConfiguration/ResourceNameTextField.tsx

```diff
@@ -5,17 +5,43 @@ import React from 'react';
 import { ITextFieldProps, TextField } from 'office-ui-fabric-react';
 import formatMessage from 'format-message';
 
+import { CheckWebAppNameAvailability } from '../../api';
+import { useDebounce } from '../useDebounce';
+
 type Props = {
-  onHostNameChange: (hostName: string) => void;
+  onHostNameChange: (hostName: string, hasErrors: boolean) => void;
```

Comment:
this is a weird callback, that callback is about a value changing not it having errors.

Reply:
We need to let  the wizard know, if there are any validation errors so we can disable the 'Next' button.

Reply:
use a dedicated callback for it then, e.g., onchange is for change, onvalidate is for validation